### DISCUSSION TITLE
콕찌르기 API 호출 횟수 확인 질문

### DISCUSSION THREAD

**Discussion by rbgus2002** [rbgus2002/cactusdy#208] [Q&A]
사용자들이 콕찌르기 기능을 얼마나 사용하는지 궁금하다고 하셔서 api 호출 횟수 count 할 수 있도록 만들어달라 하신 게 기억 나서 좀 찾아봤습니다.

아래 사진은 firebase console인데 아래 지표로는 확인이 불가능할까요?
<img width="1506" alt="image" src="https://github.com/rbgus2002/studygroup/assets/62997391/68f65d66-69b9-4fed-936e-6e6fbeaf1f30">
https://console.firebase.google.com/project/studygroup-8d056/analytics/app/ios:com.ssu.groupstudy/overview/reports~2Fexplorer%3Fparams%3D_r.explorerCard..selmet%253D%255B%2522eventCount%2522%255D%2526_r.explorerCard..seldim%253D%255B%2522eventName%2522%255D&r%3Dtop-events&fpn%3D295774991714?hl=ko

또한 구체적으로 어떤 정보가 궁금하신지, 추가적으로 궁금한 호출 정보가 있으신지 알려주세요!

**Comment by Arkady1356** [comment]
와우 원했던 통계야 ㅋㅋㅋ 굿굿
근데 대체 콜롬비아..? 에서는 찌르기를 어케한거지
